test/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerServiceSpec.scala

```diff
@@ -977,7 +967,7 @@ class ClaimTransformerServiceSpec extends AnyWordSpec with Matchers with MockFac
         val acc14                                                            = getAcc14Response()
         val displayDeclaration                                               = sample[DisplayDeclaration].copy(displayResponseDetail = acc14)
         val claim                                                            = getClaimAmounts()
-        val completeClaimsAnswer                                             = sample[CompleteClaimsAnswer].copy(List(claim))
+        val completeClaimsAnswer                                             = ClaimsAnswer(claim)
```

Comment:
might be worth renaming this as we don't have this incomplete/complete concept anymore

Reply:
done